rust/connlib/clients/apple/build-rust.sh

```diff
@@ -77,9 +84,6 @@ if [[ -n "$CONNLIB_TARGET_DIR" ]]; then
   set +x
 fi
 
-for target in "${TARGETS[@]}"
-do
-  set -x
-  cargo build --verbose --target=$target $CONFIGURATION_ARGS
-  set +x
-done
+set -x
+cargo build --verbose $TARGETS $CONFIGURATION_ARGS
```

Comment:
Cargo supports building multiple targets at once so the loop here could be removed.